corehq/apps/reports/standard/deployments.py

```diff
@@ -554,7 +555,15 @@ class ApplicationErrorReport(GenericTabularReport, ProjectReport):
 
     @classmethod
     def show_in_navigation(cls, domain=None, project=None, user=None):
-        return user and toggles.APPLICATION_ERROR_REPORT.enabled(user.username)
+        domain_access = (
+            domain_has_privilege(domain, privileges.APPLICATION_ERROR_REPORT)
+            if domain else False
+        )
+        user_access = (
+            user.is_superuser or user.is_dimagi
+            if user else False
+        )
+        return user and (domain_access or user_access)
```

Comment:
Should we do another check for the user here if we only care about if the domain has access?